frontend/resourceadm/utils/resourceUtils/resourceUtils.ts

```diff
@@ -133,3 +133,19 @@ export const mapKeywordsArrayToString = (resourceKeywords: ResourceKeyword[]): s
 export const mapKeywordStringToKeywordTypeArray = (keywrodString: string): ResourceKeyword[] => {
   return keywrodString.split(', ').map((val) => ({ language: 'nb', word: val.trim() }));
 };
+
+/**
+ * Gets the status for if a tab is active or not based on the
+ * current page and the tabs id.
+ *
+ * @param currentPage the currently selected tab
+ * @param tabId the id of the tab to check
+ *
+ * @returns if the tab is active or not
+ */
+export const getIsActiveTab = (currentPage: NavigationBarPage, tabId: number): boolean => {
+  if (currentPage === 'about' && tabId === 0) return true;
+  if (currentPage === 'policy' && tabId === 1) return true;
+  if (currentPage === 'deploy' && tabId === 2) return true;
+  return false;
```

Comment:
Hva tenker du? Ville koden bli lettere å lese/-og debugge, dersom vi ikke måtte forholde oss til en index (0-2)? Kunne ikke identifikatoren per side være av typen NavigationBarPage? 🤔 Da hadde metoden å sjekke om en tab er active noe sånn som dette:

```suggestion
export const getIsActiveTab = (currentPage: NavigationBarPage, tabId: NavigationBarPage): boolean => {
  return currentPage === tabId);
 }
```

Reply:
Problemet er at vi ønsker å gjøre selve ```LeftNavigationBar``` så gjenbrukbar som mulig, og da kan vi ikke mikse inn resourceadm sin ```NavigationBarPage``` type. 
Jeg tenker at det er mulig å gjøre ```tabId``` om fra number til string, også sjekke om de har samme verdi, men at vi ikke kan sette ```NavigationBarPage``` type på ```LeftNavigationTab``` typen for da mikser vi plutselig resourceadm og apps.